### ISSUE TITLE
Fixed loss parameters

### ISSUE THREAD

**Issue by nnt296** [zylo117/Yet-Another-EfficientDet-Pytorch#236]
Hi @zylo117, thanks for the great work. However, I have a small question regarding your FocalLoss implementation. 

Inferred from your [loss.py](https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/master/efficientdet/loss.py#L33) , I see that you are training with gamma = 2.0. Yet, you set the default gamm in [train.py](https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/master/train.py#L48) to 1.5 (which is the same as in the paper). So I am wondering which was the value you used when training your models.

```
    def forward(self, classifications, regressions, anchors, annotations, **kwargs):
        alpha = 0.25
        gamma = 2.0
        batch_size = classifications.shape[0]
        classification_losses = []
        regression_losses = []
```

```
    parser.add_argument('--alpha', type=float, default=0.25)
    parser.add_argument('--gamma', type=float, default=1.5)
    parser.add_argument('--num_epochs', type=int, default=500)
```

**Comment by zylo117** [comment]
I should remove this now.
```
parser.add_argument('--alpha', type=float, default=0.25)
parser.add_argument('--gamma', type=float, default=1.5)
```

**Comment by zylo117** [comment]
done. https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/commit/2817a547cec0a97e849db79a24317eae64f31519
